tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -114,82 +370,608 @@ To see a list of image families:
+
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+1. Wait a few minutes until the Windows VM has completed the boot up process.
+
+1. Using the [Chrome RDP plugin](https://cloud.google.com/compute/docs/instances/›connecting-to-instance#chrome-remote-desktop) or your preferred [Remote Desktop client](https://cloud.google.com/compute/docs/instances/connecting-to-instance#windows), RDP into your Windows instance.
+
+1. If you need to generate a Windows password, follow [these instructions](https://cloud.google.com/compute/docs/instances/windows/creating-passwords-for-windows-instances#generating_a_password).
+
+1. Once you've connected to the instance, verify that **git**, **python** and **7zip** has been installed successfully (matching versions we defined in `packages.config` XML manifest).
+
+![verifying packer windows build in cmd](windows/task12-windows-verify.png)
+
+</details>
+
+# Task 13: Cleaning up
+If you don't want to keep the resources after this tutorial, you can delete them by following these instructions.
+
+## Building a Linux Image
+
+<details>
+<summary>Click to reveal commands</summary>
 
 1.  Delete the firewall rule, the instance, and the image:
 
-        gcloud compute firewall-rules delete --quiet http
-        gcloud compute instances delete --quiet helloworld-from-factory
-        gcloud compute images delete --quiet helloworld-v01
+    ```sh
+    gcloud compute firewall-rules delete --quiet http
+    gcloud compute instances delete --quiet helloworld-from-factory
+    gcloud compute images delete --quiet helloworld-v01
+    ```
+
+2.  Delete the packer Cloud Build image:
+
+    ```sh
+    gcloud container images delete --quiet gcr.io/$PROJECT/packer  --force-delete-tags
+    ```
+
+3.  Delete the source repository:
+
+    ```sh
+    gcloud source repos delete --quiet helloworld-image-factory
+    ```
+
+    Only do this if you don't want to perform the tutorial in this project again. The repository name won't be usable
+    again for up to seven days.
+
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+1.  Delete the firewall rule, the instance, and the image.
 
 2.  Delete the packer Cloud Build image:
 
-        gcloud container images delete --quiet gcr.io/$PROJECT/packer  --force-delete-tags
+    ```sh
+    cloud container images delete --quiet gcr.io/$PROJECT/packer  --force-delete-tags
+    ```
 
 3.  Delete the source repository:
 
-        gcloud source repos delete --quiet helloworld-image-factory
-        
+    ```sh
+    gcloud source repos delete --quiet windows-image-factory
+    ```
+
     Only do this if you don't want to perform the tutorial in this project again. The repository name won't be usable
     again for up to seven days.
+
+</details>
+
+<br />
+<br />
+
+# Windows-packer-scripts
+
+#### cloudbuild.yaml
+```
```

Comment:
Instead of duplicating the code, you can probably use the Github widget to include it

Reply:
No native way to embed code into markdown on GH; updated this to add links to scripts vs pasting the respective code in its entirety to reduce length of tutorial page